specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-07-19/quota.json

```diff
@@ -0,0 +1,1270 @@
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			},
+			"type": "oauth2",
+			"description": "Azure Active Directory OAuth2 Flow"
+		}
+	},
+	"paths": {
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Capacity/providers/{providerId}/locations/{location}/serviceLimits/{resourceName}": {
+			"get": {
+				"tags": [
+					"QuotaInformation"
+				],
+				"summary": "Gets the current quota limit and usages for the resource provider for the specified location for the specific resource in the parameter.",
+				"description": "This API gets the current quota limit and usages for the specific resource for resource provider for the specified location. This response can be used to submit quotaRequests.",
+				"operationId": "Quota_listStatus",
+				"x-ms-examples": {
+					"Quotas_Request_ForCompute": {
+						"$ref": "./examples/getComputeOneSkuUsages.json"
+					}
+				},
+				"parameters": [
+					{
+						"$ref": "#/parameters/SubscriptionIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ProviderIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/LocationIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"$ref": "#/parameters/ResourceNameInParameters"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK. Quota information will be returned as response with pagination.",
+						"schema": {
+							"$ref": "#/definitions/CurrentQuotaLimitBase"
+						},
+						"headers": {
+							"ETag": {
+								"description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+								"type": "string"
+							}
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "#/definitions/ExceptionResponse"
+						}
+					}
+				}
+			},
+			"put": {
+				"tags": [
+					"QuotaRequests"
+				],
+				"summary": "Submits a Quota Request for a resource provider at the specified location for the specific resource in the parameter.",
+				"description": "Submits Quota change request for a resource provider for the specified location for the specific resource in the parameter. To use, first make a Get request to get quota information. This information consists of a list of resources and information regarding those resources. For all the resources in that list which requires an update to their quotas, update their limit fields in the response from the Get request to their new values. Then, submit individual request for each resource using the updated JSON object to this quota request API. This will update the quotas to the values specified. The AzureAsyncOperation header in the response will be used to track the status of the quota request. Please poll the url provided in AzureAsyncOperation Header to get the status. ",
+				"operationId": "QuotaRequest_Create",
```

Comment:
`Quotas_Create` so that all operations under this path are grouped together.

Reply:
Updated as suggested.